lib/mayaHydra/hydraExtensions/CMakeLists.txt

```
@@ -30,31 +30,54 @@ set(HEADERS
 # -----------------------------------------------------------------------------
 # compiler configuration
 # -----------------------------------------------------------------------------
-target_compile_definitions(${TARGET_NAME} 
-    PRIVATE 
-        MAYAHYDRALIB_EXPORT
-        $<$<BOOL:${IS_MACOSX}>:OSMac_>
-        # Copy-pasted from lib/mayaUsd/CMakeLists.txt
-)
-
+if(DEFINED MAYAUSD_LOCATION)
```

I don't think you need this conditional, could simply be
```suggestion
$<$<BOOL:${MAYAUSD_LOCATION}>:MAYAHYDRALIB_MAYAUSD_ENABLED>
```